import.py

```diff
@@ -120,22 +120,26 @@ def main():
 
 
 
-def reprojectShapefile(f, inputDir, outputDir, srs):
+#NB: this now dissolves shapes and reprojects.  Should I rename?
```

Comment:
Yes. I would call it something like processShapefile and put exactly what it does in some triple-quote comments above it.